float32 governor_value
bool is_estopped
